src/net_processing.cpp

```diff
@@ -3911,7 +3912,8 @@ bool PeerLogicValidation::SendMessages(CNode* pto)
                         if (!txinfo.tx) {
                             continue;
                         }
-                        if (filterrate && txinfo.feeRate.GetFeePerK() < filterrate) {
+                        if (filterrate > CFeeRate(0) &&
```

Comment:
1. Some code comments would be great, particularly before the conditional here and line 3859 just above.

2. With these changes the 2 conditionals appear to be equivalent now. Making them the same or extracting them to a well-named bool function might be a nice touch.

```cpp
                        if (filterrate > CFeeRate(0)) {
                            if (txinfo.fee < filterrate.GetFee(txinfo.vsize)) {
                                continue;
                            }
                        }
```
```cpp
                        if (filterrate > CFeeRate(0) &&
                            txinfo.fee < filterrate.GetFee(txinfo.vsize)) {
                            continue;
                        }
```

Reply:
Can you suggest some text? I'm not sure of the confusion.

Also I'm not sure what you mean by (2)

Reply:
ah I see now, I'm just mimicking current style. If I need to rebase or something I'll make them match.